CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 
 ### Changed
 
+- ([#412](https://github.com/demos-europe/demosplan-ui/pull/412)) Adjust dpValidate directive in DpMultiselect ([@ahmad-demos](https://github.com/@ahmad-demos))
```

Comment:
I think this should go under "Fixed" and we should make the message more specific, like "Import missing directive in DpMultiselect"

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/412/commits/d82b2f90e7ffc2c3abd404eaa64b4d21d422a02a